scripts/pathseq/wdl/pathseq_pipeline.wdl

```diff
@@ -0,0 +1,199 @@
+	Int machine_mem = if defined(mem_gb) then mem_gb *1000 else default_ram_mb
+	Int command_mem = machine_mem - 4000
+
+	Float default_min_score_identity = 0.9
+	Float default_identity_margin = 0.02
+
+command <<<
+    	set -e
+    	export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk4_jar_override}
+        gatk --java-options "-Xmx${command_mem}m" \
+        PathSeqPipelineSpark \
+        --input ${input_bam} \
+        --output ${bam_output_path} \
+        --scores-output ${scores_output_path} \
+        --filter-metrics ${filter_metrics_output_path} \
+        --score-metrics ${score_metrics_output_path} \
+        --kmer-file ${kmer_file} \
+        --filter-bwa-image ${filter_bwa_image} \
+        --microbe-bwa-image ${microbe_bwa_image} \
+        --microbe-fasta ${microbe_fasta} \
+        --taxonomy-file ${taxonomy_file} \
+        --bam-partition-size ${select_first([bam_partition_size, 4000000])} \
+        --is-host-aligned ${is_host_aligned} \
+        --skip-quality-filters ${select_first([skip_quality_filters, false])} \
+      	--min-clipped-read-length ${select_first([min_clipped_read_length, 60])} \
+      	--filter-bwa-seed-length ${select_first([filter_bwa_seed_length, 19])} \
+        --host-min-identity ${select_first([host_min_identity, 30])} \
+        --filter-duplicates ${select_first([filter_duplicates, true])} \
+        --skip-pre-bwa-repartition ${select_first([skip_pre_bwa_repartition, false])} \
+      	--min-score-identity ${select_first([min_score_identity, default_min_score_identity])} \
+        --identity-margin ${select_first([identity_margin, default_identity_margin])} \
+        --divide-by-genome-length ${select_first([divide_by_genome_length, true])}
+>>>
+  runtime {
+	docker: gatk_docker
+	memory: machine_mem + " MB"
+	# Note that the space before SSD and HDD should be included.
+	disks: "local-disk " + select_first([disk_space_gb, default_disk_space_gb]) + if use_ssd then " SSD" else " HDD"
+	preemptible: select_first([preemptible_attempts, 3])
+	cpu: select_first([cpu, 32])
+  }
+  output {
+    File bam_output_pathFile = "${sample_name}.pathseq.bam"
+    File outputScoresFile = "${sample_name}.pathseq.tsv"
+    File outputFilterMetricsFile = "${sample_name}.pathseq.filter_metrics"
+    File outputScoreMetricsFile = "${sample_name}.pathseq.score_metrics"
+  }
+}
+
+workflow PathSeqPipelineWorkflow {
+
+    String sample_name
+    File input_bam
+
+    File kmer_file
+    File filter_bwa_image
+    File microbe_bwa_image
+    File microbe_fasta
+    File microbe_fasta_dict
+    File taxonomy_file
+
+    Boolean is_host_aligned
+    Boolean? filter_duplicates
+    Boolean? skip_pre_bwa_repartition
+    Boolean? divide_by_genome_length
+    Int? filter_bwa_seed_length
+    Int? host_min_identity
+    Int? min_clipped_read_length
+    Int? bam_partition_size
+    Float? min_score_identity
+    Float? identity_margin
+
+	File? gatk4_jar_override
```

Comment:
Can you de-dent to make these line up with the parameters above?

Reply:
Cleaned up indentation (double-spaces everywhere)